playbooks/vagrant-devstack.yml

```diff
@@ -28,7 +28,7 @@
     - mysql
     - edxlocal
     - memcache
-    - mongo
+    - mongo_3_2
```

Comment:
Sanity check: What was the previous mongo version? Why don't we care about upgrading the mongo version and assess the impact?

Sorry, these are questions from someone without context rather than questions around the work you did, but I would appreciate it if you could share the context.

Reply:
> What was the previous mongo version?

The previous version was MongoDB 2.6.x that was deprecated a while ago and replaced in production to 3.2.x, but for some reason, it wasn't updated in the devstack.

> Why don't we care about upgrading the mongo version

Since it is an upgrade in a way, we care about upgrading MongoDB, but again, we are just catching up with the production version here.

PS: We cannot use 2.6.x anymore as the download server, on MongoDB's side, is dismantled and not available anymore.